src/features/tutorApplication/ui/subjectForm/subjectFormItem/profileCategories/index.tsx

```diff
@@ -47,13 +47,16 @@ const ProfileCategories = ({
       </div>
       <div className={styles['container__options--price']}>
         <Input
+          variant="price"
           name="price"
           value={values.price}
           onChange={handlePriceChange}
           label="Цена"
-          placeholder="2500"
-          type="number"
-          style={{ inlineSize: '97px' }}
+          placeholder={category.price}
+          type="text"
+          extraClass={styles['container__options--price-input']}
+          maxLength={6}
```

Comment:
не многовато? ))))))
```suggestion
          maxLength={5}
```

Reply:
Вдруг кто-то будет новую архитектуру написания кода преподавать)